javascript/builtins/Object.json

```
@@ -819,7 +819,8 @@
                 "version_added": "4"
               },
               "ie": {
-                "version_added": "8"
+                "version_added": "9",
+                "notes": "Also supported in Internet Explorer 8, but only on DOM objects and with some non-standard behaviors."
```

I think it would help to explain **what** the non-standard behaviors were, rather than just saying there were some.  The more details, the better!

```suggestion
                "notes": "Support in Internet Explorer 8 was only present on DOM objects with some non-standard behaviors [Add Explanation].  This behavior was fixed in Internet Explorer 9."
```

---

I have no idea what they are. I copied this phrasing from `defineProperty`.

---

Hmm...  I guess we’ll have to look into what that behavior is, then, since it’s not too helpful to mention there **is** non-standard behavior and not explain what that behavior is.  I‘ll see if I can dig into it!